### PR TITLE
Small fix in Markdown format

### PR DESCRIPTION
**Pull request by valery1707**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Combine this with the power of Angular and you've got yourself an image uploadin
 bower install angular-uploadcare
 ```
 
-#Usage
+### Usage
```
Comment (Zmoki) [inline README.md]: it's first level header, not third. keep just space.

File: README.md [hunk 2]
```diff
@@ -47,7 +47,7 @@ bower install angular-uploadcare
  />
  ```
 
-#Options
+### Options
```
Comment (Zmoki) [inline README.md]: it's first level header, not third. keep just space.

**Review by Zmoki — CHANGES_REQUESTED** [review]

**Comment by valery1707** [comment]
@Zmoki fixed

**Review by Zmoki — APPROVED** [review]

**Comment by Zmoki** [comment]
@valery1707 thank you!

**Comment by valery1707** [comment]
@Zmoki will you merge this changes?

**Comment by Zmoki** [comment]
@valery1707 sure. I waited for the tests to pass

**Pull request merged**